docs/user-guide/user-roadmap-zowe-cli copy.md

```diff
@@ -0,0 +1,161 @@
+# Installing Zowe CLI overview
+
+The resources here provide information about various Zowe CLI topics, such as learning basic skills, installation, developing, and troubleshooting.
+
+:::tip
+
+To identify the resources most relevant for you, use the following definitions of Zowe CLI skill levels.
+
+- **Beginner:** You're starting out and want to learn the fundamentals.
+- **Intermediate:** You have some experience but want to learn more in-depth skills. 
+- **Advanced:** You have lots of experience and are looking to learn about specialized topics.
+
+:::
+
+## Fundamentals
+
+> Zowe skill level: Beginner
+
+- [**Zowe CLI overview**](../getting-started/overview.md#zowe-cli)
+
+   New to Zowe CLI? This overview topic provides an introduction to Zowe CLI.
+
+- [**Architecture**](../getting-started/zowe-architecture.md#zowe-architecture)
+
+   Review the Zowe architecture to understand how Zowe CLI works in the Zowe framework.
+
+- [**Zowe CLI FAQs**](../getting-started/zowe_faq.md#zowe-cli-faq)
+
+   If you have a question, check out the FAQ, which answers the most commonly asked questions about Zowe CLI.
+
+## Quick start
+
+> Zowe skill level: Beginner
+
+- [**Zowe CLI quick start**](../getting-started/cli-getting-started.md)
+
+   Get started with Zowe CLI quickly and easily.
+
+- [**Blog: Getting Started with Zowe CLI**](https://medium.com/zowe/getting-started-with-zowe-cli-7a29420c6be7)
+
+   This blog explains how to get started with Zowe CLI quickly.
```

Comment:
We should replace this blog for various reasons.

This one might be a suitable replacement:
- https://medium.com/zowe/zowe-cli-team-config-101-be57345ed668

Reply:
Chatted w/ @adam-wolfe about this before and we agreed to remove this blog completely. (It's been removed from the non-copy version of the user-roadmap-zowe-cli.md file.) I'm happy to add this blog as a replacement -- what do you think, Adam?

Reply:
After our stand-up discussion on this, we will not be adding this blog here at this time.